### PR TITLE
Marketplace widget

### PR DESCRIPTION
**Pull request by moskalyk**

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+import '@0xsequence/design-system/styles.css'
+export const projectAccessKey = 'AQAAAAAAAEGvyZiWA9FMslYeG_yayXaHnSI'
+ 
+const chains = getDefaultChains() // optionally, supply an array of chain ID's getDefaultChains([1,137])
+const transports = Object.fromEntries(chains.map((chain: any) => [chain.id, http()]))
+ 
+// works locally on http://localhost:4444
+const waasConfigKey = 'eyJwcm9qZWN0SWQiOjE2ODE1LCJlbWFpbFJlZ2lvbiI6ImNhLWNlbnRyYWwtMSIsImVtYWlsQ2xpZW50SWQiOiI2N2V2NXVvc3ZxMzVmcGI2OXI3NnJoYnVoIiwicnBjU2VydmVyIjoiaHR0cHM6Ly93YWFzLnNlcXVlbmNlLmFwcCJ9' 
+const googleClientId = '970987756660-35a6tc48hvi8cev9cnknp0iugv9poa23.apps.googleusercontent.com'
```
Comment (taylanpince) [inline src/config.ts]: Not sure why this is overriding the keys from `.env` – I spent a bunch of time trying to figure out why the preview URL was not working even though it was whitelisted. Can you please switch it back to using `process.env`?

**Review by taylanpince — CHANGES_REQUESTED** [review]

File: src/config.ts (hunk 1 above)
Comment (moskalyk) [reply src/config.ts]: okay done

**Comment by taylanpince** [comment]
@moskalyk Thanks for updating this to use .env config. I could now test it on the deploy preview with email login. A few feedback items:

1. I don't think we need the Mint and On-ramp tabs. Let's focus on Market (Buy) and Sell.
2. What are the items listed with, and what's the price of each item? I don't see this in the Market tab.
3. Clicking on Purchase crashes the app, this should trigger the purchase flow
4. Can you set a default expiry on the Sell interface? Can be a 1 month later
5. Create Sell Order similarly fails (same error as Purchase Uncaught TypeError: Cannot read properties of undefined (reading 'token'))

**Comment by moskalyk** [comment]
1. *on-ramp*: So I think we definitely need the on-ramp, because if you're using an embedded wallet, you'll need a way to pay for your transactions. then this gives the added benefit of also on-ramping with a stable currency, so there's much less risk in transactions versus a custom currency. the other option in terms of native fees is doing a custom marketplace wrapper contract that is sponsored, but I don't think most will do this

minting: i'll remove the mint

2. not sure why, but prices are showing for me, only the second time. will try to make it more stable: 
image

3. will look into this

4. sure I'll default to 1-month

5. will look into this

**Comment by bunnybones1** [comment]
There's a flow change in the design, and we have a simpler SDK to use, so we're closing this PR

**Pull request closed without merging**